public/stylesheets/main.css

```diff
@@ -33,7 +33,7 @@
 
 .page-content {
     padding: 30px;
-    margin: 0 auto;
+    margin-left: 80%;
```

Comment:
on the uploaded page the list was looking funny if you had a bunch of uploads.

buuut this wasn't the right way to do it so there's a more specific selector used now